sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormTrainingClientTestBase.java

```diff
@@ -85,9 +109,8 @@ private static void validateErrorData(List<ErrorInformation> expectedErrors,
         }
     }
 
-    static void validateAccountProperties(AccountProperties expectedAccountProperties,
-        AccountProperties actualAccountProperties) {
-        assertEquals(expectedAccountProperties.getCustomModelLimit(), actualAccountProperties.getCustomModelLimit());
+    static void validateAccountProperties(AccountProperties actualAccountProperties) {
+        assertNotNull(actualAccountProperties.getCustomModelLimit());
```

Comment:
Could you put a TODO note with the created isssue to track the inconsistent behavior we were seeing.

Reply:
we should not check the exact number but not null should be the better way. Create a bug issue: #9269